XnViewMP.yaml

```diff
@@ -0,0 +1,72 @@
+name: XnViewMP
+buildsystem: simple
+sources:
+  - type: archive
```

Comment:
It should use extra-data instead, as we're now indirectly violating their license by using test builds.

Reply:
https://github.com/flathub/flathub/pull/883#discussion_r263404165

@alexlarsson said:
> Yeah, i think avoiding extra data when not needed due to licensing issues is a much better experience.